dirty_cat/datasets/datasets.py

```diff
@@ -0,0 +1,24 @@
+import pandas as pd
+import os
+
+# TODO: speficy pandas in dependencies
+
+FOLDER_PATH= os.path.dirname(os.path.realpath(__file__))
+print(FOLDER_PATH)
+DATA_NAME = 'midwest_survey.csv'
+
+
+def fetch_data():
+    """
+    function that fetches example data
+    - either from the internet
+    - or locally
+    because the base example dataset (midwest survey) chosen is very small,
+    it is locally stored in data/midwest_survey.csv
+    in the future, for bigger dataset, this function will get the data
```

Comment:
I'd rather have one function per dataset in the future (like in scikit-learn).

Reply:
Actually, I think that I would rather return the filename and use pandas in the example to load. It's more didactic.